releasenotes/notes/curve-analysis-02a702a81e014adf.yaml

```
@@ -0,0 +1,71 @@
+---
+features:
+  - |
+    :class:`CurveAnalysis` delegates the core fitting functionality to the LMFIT package.
```

sphinx links need a leading `.` to work without the full path. LMFIT should be updated to be a link

```suggestion
    The :class:`.CurveAnalysis` has been modified to delegate the core fitting functionality to the
    `LMFIT <https://lmfit.github.io/lmfit-py/>`_ package. Fit curves are specified using
    `LMFIT Model <https://lmfit.github.io/lmfit-py/model.html#>`_ objects. For multi-curve fitting
    a list of models can be used.
    
    A new analysis option ``fit_method`` has been added to allow a user to select the fitting algorithm used
    by the LMFit `minimizer <https://lmfit.github.io/lmfit-py/fitting.html>`_.
    The default fit method is ``"least_squares"``.
```